### ISSUE TITLE
[SCROLLBACK] Rendering not fixed after initial resize for font metrics.

### ISSUE THREAD

**Issue by Sas-18** [jwilm/alacritty#1282]
I don't believe this is a duplicate of an open issue, although very similar. If it turns out to be covered by (or the same as) one or another of the issues I've linked below, I can close this (or someone can move it?) since as mentioned elsewhere, this may be a known race condition that hasn't been pinned down yet.

Please compare these links:
[Not quite the same as issue 459](https://github.com/jwilm/alacritty/issues/459).
[Similar to issue 699 but I don't need maximized to get the bug](https://github.com/jwilm/alacritty/issues/699) and [somewhat more similar to issue 540 though I use Metacity but that shouldn't matter](https://github.com/jwilm/alacritty/issues/540).

**OS**: Linux Mint 18.3 Sylvia (Ubuntu 16.04)
**YES**: `metacity --no-composite` + X11 + Xinerama + Nvidia 384.111
**NO**: Composite nor RandR nor Wayland

Branch I'm reporting this for: **Scrollback**
but right now also rebuilding master at the moment to see if I can repro it there too.
**UPDATE**: rebuild master done, 30+ attempts so far, no reproduce with master branch...

**Issue Describe**:
It doesn't happen every time, or even in 20 times, but eventually it will happen. I started _Alacritty-scrollback_ from another terminal 20 or 30 times to repro this; once it's open and ready to use, it is not rendered properly after the resize adjustment for font metrics.
#In photos below; first is how _Alacritty-scrollback_ looks when it opens and presents the bug. Second photo shows after I hit _ENTER_, it fixes the render.
The window is focused (freshly opened, focused) when this all is occur.

Photo to compare before _ENTER_ and after _ENTER_:
![screenshot from 2018-04-29 06-07-41](https://user-images.githubusercontent.com/38713922/39405157-65452d36-4b76-11e8-8d1b-565a6966ad54.png)
![screenshot from 2018-04-29 06-08-42](https://user-images.githubusercontent.com/38713922/39405158-655ba624-4b76-11e8-85f8-88494ca3ff31.png)

**Comment by chrisduerr** [comment]
I can't reproduce it but it doesn't look like an issue caused by scrollback but rather like something caused by events not being handled properly.

**Comment by chrisduerr** [comment]
@Sas-18 could you share the full output of `alacritty -vvv` to help troubleshoot this maybe? I'd be interested in getting the output on master, scrollback when it works and scrollback when it's broken. It's probably best to just do the minimum necessary to make this show up, to prevent any noise of the output.

**Comment by chrisduerr** [comment]
@Sas-18 I'd be interested in knowing if this issue is solved by moving the mouse over the window triggering additional redraw events. This might be related to #824, #1227 and others.

If nobody else can reproduce this I'm probably just going to assume this will be solved once the other issues are fixed.

**Comment by chrisduerr** [comment]
I have a hunch that this issue might be solved with #1518, since another similar issue was fixed with it. I've added this issue to the issues which will be automatically closed once that PR is merged.

If anyone is able to reproduce this issue with #1518, please let me know and I'll remove the reference to this issue.
